### PR TITLE
DOC-7643: Document how to manage teams via the System Console

### LINKED ISSUE #7643: Document how to manage teams via the System Console
## Context

Details on how to manage Mattermost teams as a system admin using the System Console are missing from the Mattermost Product Documentation.

## Docs Request

Develop admin documentation for the following topics:

- Find a team
- Filter team searches
- Review team configuration
- Manage team membership
  - Add members
  - Remove members
  - Sync group members
  - Anyone can join
  - Only specific email domains can join
- Archive a team

**Note**: The ability to add groups to a team also needs to be documented, but requires additional internal investigation/testing first.

## How you can help

- Create a single docs PR that contains all of these changes across as many commits as necessary
- Use the latest Mattermost mobile app and the standard Mattermost Denim theme to capture all workflows.
- Use existing admin docs content as a starting point for voice and tone guidance. The Docs team will review and provide support for publication.

### PR DESCRIPTION
**Pull request by Rutam21**
## Summary

Detailed steps for the team administration workflows were required but absent. So, this PR aims to add the steps for all those workflows for teams management on the [User management configuration settings](https://docs.mattermost.com/configure/user-management-configuration-settings.html#teams) page. 

The following workflows have been added.

- Find a team
- Filter team searches
- Review team configuration
- Manage team membership
    - Add members
    - Remove members
    - Sync group members
    - Anyone can join
    - Only specific email domains can join
- Archive a team

## Ticket Link

This PR fixes #7643  

## Checklist

- [x] This PR follows the Mattermost Contributing Guidelines.
- [x] Relevant product screenshots in the Denim theme have been added.
- [x]  Workflows are structured as a numbered list for the steps instead of paragraphs.
- [x] The changes are tested using a local build.

### CODE REVIEW

**Comment by Rutam21** [comment]
@cwarnermm Please review and suggest changes, if any. Thanks.

**Comment by Rutam21** [comment]
@cwarnermm I have updated the PR with the suggested changes in #7654. Please review and suggest changes, if any. Thanks.

File: source/configure/user-management-configuration-settings.rst [hunk 1]
```diff
@@ -255,26 +253,129 @@ Teams
...
+2. Enter the team name in the **Search** box.
+
+.. image:: ../images/find-teams.png
+  :alt: Find a Mattermost team using the System Console.
+
+.. tip::
+
+  From Mattermost v9.6, you can:
+
+  - Search for specific teams by entering a partial or full team name in the **Search** field and pressing :kbd:`Enter`.
```
Comment (cwarnermm) [inline source/configure/user-management-configuration-settings.rst:271]:
```suggestion
  From Mattermost v9.6, you can search for specific teams by entering a partial or full team name in the **Search** field and pressing :kbd:`Enter`.
```

File: source/configure/user-management-configuration-settings.rst [hunk 2]
```diff
@@ -255,26 +253,129 @@ Teams
...
+
+Filter team searches
+~~~~~~~~~~~~~~~~~~~~
+
+Filter your team search to narrow down results based on the team management type (anyone can join, invite only or group sync).
```
Comment (cwarnermm) [inline source/configure/user-management-configuration-settings.rst:276]:
```suggestion
Filter your team search to narrow down results based on the team management type (anyone can join, invite only, or group sync).
```

File: source/configure/user-management-configuration-settings.rst [hunk 3]
```diff
@@ -255,26 +253,129 @@ Teams
...
+2. Select the team from the list to view its configuration page.
 
-See the :ref:`archive a team <collaborate/organize-using-teams:archive a team>` documentation for details on archiving teams that are no longer needed.
+.. image:: ../images/team-configuration-details.png
+  :alt: Review team configuration details using the System Console
+
+Manage team membership
+~~~~~~~~~~~~~~~~~~~~~~
+
+Admins can directly add or remove member from the team. They can also customize how members are added to the team using the System Console.
```
Comment (cwarnermm) [inline source/configure/user-management-configuration-settings.rst:299]:
```suggestion
Admins can directly add or remove member from the team and customize how members are added to the team using the System Console.
```

**Review by cwarnermm — APPROVED** [review]
Thanks, @Rutam21!

**Pull request merged**